tests/test_worksheet.py

```diff
@@ -5,7 +5,7 @@
 from oar.core.const import *
 from oar.core.exceptions import WorksheetException
 from oar.core.worksheet import WorksheetManager
-
+from oar.core.worksheet import TestReport
```

Comment:
Is it a plan to add test? Have you tried to check the link values by the api?

If it is not possible to test the cell, we don't need the import.